generate/generate_nirspec.py

```diff
@@ -376,31 +376,35 @@ def reorder(pcfName, verbose=False):
             print ('Order', order, terms, ' terms')
 
         if '*xForward' in text:
-            text = pcf.readline()
-            f = text.split()  # Array of terms text strings
+            # text = pcf.readline()
+            # f = text.split()  # Array of terms text strings
             for k in range(terms):
-                xForward.append(float(f[k]))
+                text = pcf.readline()
```

Comment:
Why was this code change introduced? It is not clear to me that these two implementations are equivalent (if they are supposed to be).
I would feel more comfortable if it could be demonstrated by an independent check (on a different branch) that this code reproduces the "bugfix-only" version of the SIAF currently on the siaf-updates branch.

Reply:
You're right, this change isn't equivalent - it's there because @charlesrp changed the format of the `pcf` file so the x/y Forward/Backward coefficients are separated by new lines, rather than by spaces on a single line.